### DISCUSSION TITLE
More beneficial .  Run fishnet or donate to lichess ??

### DISCUSSION THREAD

**Discussion by jim-perkins** [lichess-org/fishnet#218] [Q&A] [answered]
Hi

I was running fishnet to help out but shut it off a couple months ago due to the energy costs.  Think I was spending > $30 per month.  I have dual xeon cpu's on gen 8 and 9 supermicro chassis, about 30 threads.  Wondered if my server was even missed as there isnt really a way to know how well the analysis is being served by volunteers , or if that is cost effective ? 

So which would be more beneficial, running the server or donating to lichess ??

thanks for your time and work on fishnet

**Comment by niklasf** [comment, marked as answer]
Hi, based on https://lichess.org/costs, it appears to be more efficient than our own instances at OVH. Thank you!

Somehow systematically collecting the costs (and maybe energy mix to estimate emissions) would be interesting, especially if the client itself could use it to prioritize or to recommend shutting down.

**Comment by jim-perkins** [comment]
Thanks for the reply.  I restarted my machine.  The Lichess End of Year Update 2021 is what motivated me.  I really like everything about the way Lichess is done.  free, open source, no ads, etc .  I play chess nearly every day.  Keep up the good work :-)

So the machine I had available is probably gonna cost me $12-$25 per month.  Fishnet doesnt utilize me 100% of the time.
When it does I'm currently averaging around 5413 knps 

Energy Charge per kWh  $.10098 (10.098 cents) .  This is mid America USA ( Tennessee )
2.9 amps is a very rough estimate of current draw of fishnet by polling current draw of APC UPS.  the UPS is handling 3 servers plus networking gear.  compared value to when fishnet was using all the threads to when fishnet was idle.

<html>
<body>
amps | 2.9
-- | --
volts | 120
watts | 348

watts | 348
-- | --
hours per day | 24
kw per day | 8.352
kw per month | 254.04
cost per month | $25.65
</body>
</html>



Supermicro X8DTU-F with Dual sockets of Intel(R) Xeon(R) CPU  X5650  @ 2.67GHz  .  KVM virt machine using 22 of available 24 threads.
Jan 05 17:38:08 lichess fishnet-x86_64-unknown-linux-gnu[1048]: ><> fishnet/2.5.1: 5413 knps (nnue), <redacted>

and never thought about it before now but my power is more than likely[ TVA Hydroelectric power ](https://en.wikipedia.org/wiki/Tennessee_Valley_Authority)
